.eslintignore

```diff
@@ -1,2 +1,3 @@
 src/js/modules/*.js
 src/js/components/*.js
+src/js/prev.js
```

Comment:
- [ ] Kindly make sure to disable the eslint ignore list as that defies the purpose of having linters in the first place. **_This can potentially invalidate your PR submission in the future._**